### ISSUE TITLE
Missing libcups.so.2 in portable build

### ISSUE THREAD

**Issue by qbitroot** [FreeTubeApp/FreeTube#1294]
**Behavior of the program**
When I run `./freetube` from freetube-0.13.0-linux-portable-x64.zip, I get this:
```
./freetube: error while loading shared libraries: libcups.so.2: cannot open shared object file: No such file or directory
```

**Expected behavior**
User is able to run FreeTube from portable build without additional library installation.

**To Reproduce**
1. Go to https://github.com/FreeTubeApp/FreeTube/releases
2. Click on Assets > freetube-0.13.0-linux-portable-x64.zip
3. Extract archive
4. Run `./freetube`
5. See missing library error.

**Environment Information (please complete the following information):**
 - OS: Gentoo GNU/Linux
 - OS Version: N/A (rolling-release); profile: amd64/17.1
 - FreeTube version: 0.13.0
 - Installation Method (If applicable): Portable, from releases page.

**Additional context**
A user has to install CUPS print server to have libcups.so.2. Why does FreeTube ever have it as a dependency?

Please include libcups.so.2 into the archive for portable program.

**Comment by qbitroot** [comment]
Found a solution: setup a debian VM with directory sharing, copy `libavahi-client.so.3, libavahi-common.so.3, libcups.so.2, libgssapi_krb5.so.2, libk5crypto.so.3, libkeyutils.so.1, libkrb5.so.3, libkrb5.so.3.3, libkrb5support.so.0, libxkbcommon.so.0` to freetube directory and run freetube with `./freetube --no-sandbox`. Works like a charm, but please, include those libraries into the official release archive.

**Comment by Svallinn** [comment]
I don't believe this is a problem from our end, since I ran bare `electron` on my machine and it reported the same error.
Would you mind trying to install `electron` on your machine, running it and checking if the same error occurs?
If it does, please close this issue and report the issue to the appropriate entity.
